### DISCUSSION TITLE
How to link to previous and next article?

### DISCUSSION THREAD

**Discussion by navgatt** [CaiJimmy/hugo-theme-stack#1108] [Q&A]
Hi all, 

is it possible - and if yes, how - to have links to the previous and next article, when visiting a blog post?

**Comment by Sortova** [comment]
Here is what I did:

Edit:

`/layouts/partials/article/article.html`

and add this at the bottom:

```
<nav>
  <hr />
  {{with .Next}}
  <div>
  <span style="display:flex; float: left; vertical-align: middle; line-height: 64px;">
    &nbsp;<a style="font-size: 20px;" href="{{.Permalink}}"><img src="/images/left-arrow.svg" /></a>&nbsp;&nbsp;<a style="font-size: 20px;" href="{{.Permalink}}">{{.Params.Title}}</a>
  {{end}}
  </span>

  {{with .Prev}}
  <span style="display:flex; float: right; vertical-align: middle; line-height: 64px;">
    <a style="font-size: 20px;" href="{{.Permalink}}">{{.Params.Title}}</a>&nbsp;&nbsp;<a style="font-size: 20px;" href="{{.Permalink}}"><img src="/images/right-arrow.svg" /></a>&nbsp;
  {{end}}
  </span>
  </div>

</nav>
  <p>
  <br />
  <br />
  <br />
  </p>

</article>
```

Note that this is something of a kludge since I didn't bother with adding the style to the stylesheet and playing with layout since my CSS skills are week, and you'll need to find left and right arrow icons you like. My icons are 64px high which is why I set the line-height to 64px.

Hope this helps and sorry for my limited skills.
